docs/guides/fullstack_app_guide.md

```diff
@@ -0,0 +1,367 @@
+# A Guide to Building a Full-Stack Web App with LLamaIndex
+
+LlamaIndex is a python library, which means that integrating it with a full-stack web application will be a little different than what you might be used to.
+
+This guide seeks to walk through the steps needed to create a basic API service written in python, and how this interacts with a TypeScript+React frontend.
+
+All code examples here are available from the [llama_index_start_pack](https://github.com/logan-markewich/llama_index_starter_pack/tree/main/flask_react) in the flask_react folder.
+
+The main technologies used in this guide are as follows:
+
+- python3.11
+- llama_index
+- flask
+- typescript
+- react
+
+## Flask Backend
+
+For this guide, our backend will use a [Flask](https://flask.palletsprojects.com/en/2.2.x/) API server to communicate with our frontend code. If you prefer, you can also easily translate this to a [FastAPI](https://fastapi.tiangolo.com/) server, or any other python server library of your choice.
+
+Setting up a server using Flask is easy. You import the package, create the app object, and then create your endpoints. Let's create a basic skeleton for the server first:
+
+```python
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def home():
+    return "Hello World!"
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=5601)
+```
+
+~flask_demo.py~

```

Comment:
this does a strikethrough on the text (not sure this was intended) 

there's a few instances below as well

Reply:
oh, it was doing subscript for me in vscode lol I will double check the syntax